src/main/java/fi/aalto/cs/apluscourses/model/Module.java

```diff
@@ -159,4 +160,19 @@ public String getVersionId() {
       return versionId;
     }
   }
+
+  //todo: remove as a duplicate (of a Scala object code that I could not call directly)
+  // ReplUtils.getReplInitialCommandsForModule(...) & test me!
+  @Nullable
+  protected static String[] getReplInitialCommandsForModule(
+      com.intellij.openapi.module.Module module) {
+    return PluginSettings
+        .getInstance()
+        .getMainViewModel(module.getProject())
+        .courseViewModel
+        .get()
+        .getModel()
+        .getCourseReplInitialCommands()
+        .getOrDefault(module.getName(), null);
```

Comment:
Nit: `module` should have `@NotNull`.

Also thinking about this method, since it accesses the `module` and `project`, it should have a read action.

I think the method could also be moved inside `IntelliJModule`